packages/react-input/src/components/Input/Input.stories.tsx

```diff
@@ -5,13 +5,14 @@ import { Input } from './Input';
 import { getNativeElementProps, useId } from '@fluentui/react-utilities';
 import { InputProps } from './Input.types';
 import { ArgTypes } from '@storybook/react';
-// prevent terrible reload times by using deep imports :(
-import Search16Regular from '@fluentui/react-icons/lib/esm/components/Search16Regular';
-import Search20Regular from '@fluentui/react-icons/lib/esm/components/Search20Regular';
-import Search24Regular from '@fluentui/react-icons/lib/esm/components/Search24Regular';
-import Dismiss16Regular from '@fluentui/react-icons/lib/esm/components/Dismiss16Regular';
-import Dismiss20Regular from '@fluentui/react-icons/lib/esm/components/Dismiss20Regular';
-import Dismiss24Regular from '@fluentui/react-icons/lib/esm/components/Dismiss24Regular';
+import {
+  Search16Regular,
```

Comment:
Do resizable icons still include the size in their name ?

Reply:
No, they don't these are the fixed size icons still. They are imported from the same package. I will probably make a separate PR to switch these icons to the resizeable ones after touching base with the controls' codeowners